src/components/screens/monitor/delegates/delegates.test.js

```diff
@@ -53,7 +53,7 @@ describe('Delegates monitor page', () => {
   });
 
   it('renders a page with header', () => {
-    expect(wrapper.find('Box header')).toIncludeText('Active delegates');
+    expect(wrapper.find('Box header').at(1)).toIncludeText('Active delegates');
```

Comment:
💅 I would rather add some class to the Box. Because this way, it will be again broken when we add the box with charts.
```suggestion
    expect(wrapper.find('Box.delegate-table header')).toIncludeText('Active delegates');
```

Reply:
✅